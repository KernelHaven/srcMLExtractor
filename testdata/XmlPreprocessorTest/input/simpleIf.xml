<unit language="C">
	<cpp:if>#<cpp:directive>if</cpp:directive><expr>A</expr></cpp:if>
	<empty_stmt>;</empty_stmt>
	<cpp:endif>#<cpp:directive>endif</cpp:directive></cpp:endif>
</unit>
