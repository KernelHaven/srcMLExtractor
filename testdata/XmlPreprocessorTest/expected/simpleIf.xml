<unit language="C">
	<cpp:if>#<cpp:directive>if</cpp:directive><expr>A</expr>
		<empty_stmt>;</empty_stmt>
	</cpp:if>
</unit>
